umbraco/Umbraco-CMS
# V15: Add logic for UrlSegments back

# Notes
- This adds back the logic for "UrlSegment" on IPublishedContent. 
- Obsolete UrlSegment telling people to use the new `IDocumentUrlService`

# How to test
- Create a document type with a template, and allow it as root, and allow itself as child node
- In your template, add `@Model.UrlSegment` somewhere, so you can go to that page, and check the URL segment.
- Create a tree structure with the document type, Root - Three children - 3 grand children
- Assert the URL segments are the same as in V13 (by doing the same steps in v13)
